fido/src/main/java/com/yubico/yubikit/fido/webauthn/PublicKeyCredential.java

```diff
@@ -76,13 +113,21 @@ public AuthenticatorResponse getResponse() {
         return response;
     }
 
+    @Nullable
+    public ClientExtensionResults getClientExtensionResults() {
+        return clientExtensionResults;
+    }
+
     public Map<String, ?> toMap(SerializationType serializationType) {
         Map<String, Object> map = new HashMap<>();
         map.put(ID, getId());
         map.put(TYPE, getType());
         map.put(RAW_ID, serializeBytes(getRawId(), serializationType));
         map.put(AUTHENTICATOR_ATTACHMENT, AuthenticatorAttachment.CROSS_PLATFORM);
         map.put(RESPONSE, getResponse().toMap(serializationType));
+        if (getClientExtensionResults() != null) {
+            map.put(CLIENT_EXTENSION_RESULTS, getClientExtensionResults().toMap());
```

Comment:
Should ClientExtensionResults.toMap() also take SerializationType?

Reply:
Added SerializationType parameter to `ClientExtensionResults.toMap()` in https://github.com/Yubico/yubikit-android/commit/7a520b53518535514e556c66938e9672fc3fb77c